### ISSUE TITLE
[1.1] No aggressive mobs spawning

### ISSUE THREAD

**Issue by DeadlyAccurate** [simibubi/Above-and-Beyond#56]
Ever since I updated to 1.1, my server hasn't seen any aggressive mobs outside of spawners. I thought maybe Exotic Birds was choking the spawns, so I quadrupled the spawnRate chance of each bird (so each one is rarer), but I'm still not seeing anything. I looked through the mod list trying to find a mod that might be controlling spawns (like In Control), but I didn't see any mod I recognized.

A correction: Phantoms are spawning, but they're the only enemy mobs to do so. I've searched the entirety of both the kubejs and config folders for any mention of phantoms, mobs, etc. trying to find anything that might be stopping them from spawning, but nothing is jumping out at me.

**Comment by DeadlyAccurate** [comment]
The problem was the Exotic Birds mod. 

I noticed that in the config, allowDespawning of non-tamed birds was set to false. But even setting it to true kept the vast majority of hostile mobs from spawning. 

My next test is to set the spawnRates of every bird to 0, and we'll only have tamed birds found from eggs.

```
#Allow non-tamed birds to naturally despawn
	allowDespawning = false
```

**Comment by DeadlyAccurate** [comment]
I wanted to update this in case anyone else has this problem,  because I've figured out exactly what the issue is, but not what mod is causing it. Now, it should be noted I've added mods to this modpack, so I'm only pointing this out in case others have this problem. It could well be one of the mods I added and have nothing to do with the modpack.

Every few in-game days, I suddenly notice hostile mobs stop spawning. I installed the mod TellMe, which can output various types of data, and had it run a report of every loaded entity on the server. When I did that, I noticed _thousands_ of Shulkers were spawned in the overworld at x=0, y=-64, z=0. If I teleport to that point, there's nothing actually there, but it clears the problem up until it happens again. I can't figure out what is causing them to spawn, though, so all I can do is clear the problem until we notice it happening again. 

(If you end up needing to do this, make sure you drop your stuff in a chest -- don't forget your backpack and jewelry! -- and plan to either store your XP or give it back to yourself, because you're going to die doing this).
